modin/core/storage_formats/pandas/utils.py

```diff
@@ -24,6 +24,8 @@ def _nullcontext(dummy_value=None):  # noqa: PR01
     """
     Act as a replacement for contextlib.nullcontext missing in older Python.
 
+    Notes
+    -----
```

Comment:
Do you need a notes section?

Reply:
@mvashishtha told me to add this section

Reply:
You mean @vnlitvinov ? Ok.

Reply:
Yes, sorry!